lib/beaker/dsl/helpers.rb

```diff
@@ -23,14 +24,12 @@ module DSL
     #
     #
     module Helpers
-      include Beaker::DSL::Helpers::FacterHelpers
       include Beaker::DSL::Helpers::HostHelpers
-      include Beaker::DSL::Helpers::PuppetHelpers
       include Beaker::DSL::Helpers::TestHelpers
-      include Beaker::DSL::Helpers::TKHelpers
```

Comment:
it looks like facter, tk and puppet helper are all going into beaker-puppet (https://github.com/puppetlabs/beaker-puppet/commit/3878c13311bf5ddcda043f5c23cb1354614a997a) does this mean beaker-facter is going away?  
i doubt beaker-facter is all that heavy, that might be a good idea.  what about beaker-hiera?